SparkPost/matchbox
# Resolves #858 Update rowSpan and colSpan proptypes

Closes #858

### What Changed
- Changes table cell `rowSpan` and `colSpan` prop types to accept both strings and numbers

### How To Test or Verify
- Run libby -> http://localhost:9001/?path=Table__all-table-components
- Verify no proptype warnings appear in the console

### PR Checklist
